db/schema.rb

```diff
@@ -45,6 +45,7 @@
     t.boolean "notification", default: false
     t.datetime "archived_at"
     t.boolean "notify_parents", default: true
+    t.string "accumulative_balance_timeframe", default: "day"
```

Comment:
This is a user option but not account.

Let's move the persistence issue to a separate task. We should create `UserAccountSettings` model and keep this kind of data there.